packages/arb-token-bridge-ui/src/util/NetworkUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { L1Network, L2Network } from '@arbitrum/sdk'
+import { hexValue } from 'ethers/lib/utils'
+import { ChainId, getNetworkName, rpcURLs } from './networks'
+import { BigNumber } from 'ethers'
+import { Web3Provider } from '@ethersproject/providers'
+import EthereumLogo from '../assets/EthereumLogo.png'
+import ArbitrumOneLogo from '../assets/ArbitrumOneLogo.svg'
+import ArbitrumNovaLogo from '../assets/ArbitrumNovaLogo.png'
+
+/* 
+  The list which will be available for network selection in navbar-dropdowns
+ */
+export const networkSelectionList = [
+  ChainId.ArbitrumOne,
+  ChainId.ArbitrumNova,
+  ChainId.Mainnet
+]
+
+/* 
+  Icons and colors attached with our networks available for selection
+*/
+export const networkStyleMap: {
```

Comment:
I'd define this in the component, it's pure styling, I don't think it fit in util file